README.md

```diff
@@ -19,35 +27,45 @@ git clone git@github.com:ops-guru/docker-hive.git
 Create image
 
 ```
-docker build . -t amzn-hive-image
+docker build . -t opsguruhub/docker-hive
 ```
 
+You may want to do some cleanup first:
+docker container stop local-hive; docker rm local-hive; docker rmi opsguruhub/docker-hive
+
 ## DockerHub
 
 Image available on DockerHub
 
 ```
-docker run -d opsguruhub/docker-hive:0.0.1
+docker pull opsguruhub/docker-hive:0.0.2
 ```
 
 ## Test
 
```

Comment:
@alex-ber I think you can define an env variable:
```
export CONTAINER_NAME="local-hive"
```
and then refer to the env variable further.

Reply:
I want to be able to copy&paste command to the command line. It is also desirable that any command is self-contained.